### ISSUE TITLE
Research mask-rcnn repository to use for projects

### ISSUE THREAD

**Issue by chuanqichen** [chuanqichen/deepcoaching#3]
https://github.com/matterport/Mask_RCNN/blob/master/samples/shapes/train_shapes.ipynb   
Tensorflow version is more popular with 11,770 stars and 5,126 Fork

https://github.com/facebookresearch/maskrcnn-benchmark   
The facebook's pytorch version has 5,108 Star and 1,180 Fork.
